Configuration/StandardSequences/python/VtxSmeared.py

```diff
@@ -38,7 +38,9 @@
     'Shifted15mmCollision2015'  :    'IOMC.EventVertexGenerators.VtxSmearedShifted15mmCollision2015_cfi',
     'NominalCollision2015'  :        'IOMC.EventVertexGenerators.VtxSmearedNominalCollision2015_cfi',
     'NominalHICollision2015':      'IOMC.EventVertexGenerators.VtxSmearedNominalHICollision2015_cfi',
-    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi'
+    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi',
+    'RealisticCentroidNominalEmittance13TeVCollision': 'IOMC.EventVertexGenerators.RealisticCentroidNominalEmittance13TeVCollision_cfi',
+
```

Comment:
ok, what I wrote was probably confusing and the fact that in the current PR I forgot by mistake to add VtxSmeared didn't help.

I am changing the name and fixing the typo, it'll eventually read
`'Realistic50ns13TeVCollision':'IOMC.EventVertexGenerators.VtxSmearedRealistic50ns13TeVCollision_cfi',`

I'll propagate the fix to the other PRs as well.